ipfscluster_test.go

```diff
@@ -2108,3 +2111,47 @@ func TestClustersFollowerMode(t *testing.T) {
 		}
 	})
 }
+
+func TestClusterPinsWithExpiration(t *testing.T) {
+	ctx := context.Background()
+
+	stateSyncInterval = 2 * time.Second
```

Comment:
Please don't use a global variable. You can actually trigger StateSync manually in this test. That is the simple option.